README.md

```diff
@@ -15,61 +15,64 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 Table of Contents
 =================
 
-   * [Major Cloud Providers' Always-Free Limits](#major-cloud-providers)
-   * [Cloud management solutions](#cloud-management-solutions)
-   * [Analytics, Events and Statistics](#analytics-events-and-statistics)
-   * [APIs, Data and ML](#apis-data-and-ml)
-   * [Artifact Repos](#artifact-repos)
-   * [BaaS](#baas)
-   * [CDN and Protection](#cdn-and-protection)
-   * [CI and CD](#ci-and-cd)
-   * [CMS](#cms)
-   * [Code Quality](#code-quality)
-   * [Code Search and Browsing](#code-search-and-browsing)
-   * [Crash and Exception Handling](#crash-and-exception-handling)
-   * [Data Visualization on Maps](#data-visualization-on-maps)
-   * [DBaaS](#dbaas)
-   * [Design and UI](#design-and-ui)
-   * [Design Inspiration](#design-inspiration)
-   * [Dev Blogging Sites](#dev-blogging-sites)
-   * [DNS](#dns)
-   * [Docker Related](#docker-related)
-   * [Domain](#domain)
-   * [Email](#email)
-   * [Font](#font)
-   * [Forms](#forms)
-   * [IaaS](#iaas)
-   * [IDE and Code Editing](#ide-and-code-editing)
-   * [International Mobile Number Verification API and SDK](#international-mobile-number-verification-api-and-sdk)
-   * [Issue Tracking and Project Management](#issue-tracking-and-project-management)
-   * [Log Management](#log-management)
-   * [Management Systems](#management-system)
-   * [Messaging and Streaming](#messaging-and-streaming)
-   * [Miscellaneous](#miscellaneous)
-   * [Monitoring](#monitoring)
-   * [PaaS](#paas)
-   * [Package Build System](#package-build-system)
-   * [Payment and Billing Integration](#payment-and-billing-integration)
-   * [Privacy Management](#privacy-management)
-   * [Screenshot APIs](#screenshot-apis)
-   * [Flutter Related and Building IOS Apps without Mac](#flutter-related-and-building-ios-apps-without-mac)
-   * [Search](#search)
-   * [Security and PKI](#security-and-pki)
-   * [Authentication, Authorization and User Management](#authentication-authorization-and-user-management)
-   * [Source Code Repos](#source-code-repos)
-   * [Storage and Media Processing](#storage-and-media-processing)
-   * [Tunneling, WebRTC, Web Socket Servers and Other Routers](#tunneling-webrtc-web-socket-servers-and-other-routers)
-   * [Testing](#testing)
-   * [Tools for Teams and Collaboration](#tools-for-teams-and-collaboration)
-   * [Translation Management](#translation-management)
-   * [Vagrant Related](#vagrant-related)
-   * [Visitor Session Recording](#visitor-session-recording)
-   * [Web Hosting](#web-hosting)
-   * [Commenting Platforms](#commenting-platforms)
-   * [Browser based hardware emulation](#browser-based-hardware-emulation-written-in-javascript)
-   * [Remote Desktop Tools](#remote-desktop-tools)
-   * [Game Development](#game-development)
-   * [Other Free Resources](#other-free-resources)
+- [free-for.dev](#free-fordev)
+- [Table of Contents](#table-of-contents)
+  - [Major Cloud Providers](#major-cloud-providers)
```

Comment:
Please restrict this to just adding what you need. We like the markdown syntax we use and general layout here.